views/reports/custom_reports_table.php

```diff
@@ -17,14 +17,16 @@
 	
 	global $db;
 
-	$month = $_POST['month'];
-	$year = $_POST['year'];
+	$startDate = $_POST['startDate'];
+	$endDate = $_POST['endDate'];
+	$sDateFormatted = date_format(date_create($startDate),"m/d/Y");
+	$eDateFormatted = date_format(date_create($endDate),"m/d/Y");
+	
 	$locs = isset($_POST['location']) ? $_POST['location'] : [];
 	$races = isset($_POST['race']) ? $_POST['race'] : [];
 	$minAge = $_POST['minAge'];
 	$maxAge = $_POST['maxAge'];
-	$monthQuery = "(date_part('month', participantclassattendance.date) = $month)";
-	$yearQuery = "(date_part('year', participantclassattendance.date) = $year)";
+	$dateQuery = "(participantclassattendance.date >= '$startDate' AND participantclassattendance.date <= '$endDate')";
 	$locQuery = "";
 	
 	if (count($locs) > 0) {
```

Comment:
`siteName` no longer exists in `participantclassattendance` the proper column to be referenced it `curriculumName`. This part should then look like:

```php
if (count($locs) > 0) {
    $locQuery = "(participantclassattendance.curriculumName = '" . $locs[0] . "' ";
    for ($i = 1; $i < count($locs); $i++) {
        $locQuery .= "OR participantclassattendance.curriculumName = '" . $locs[$i] . "' ";
    }
    $locQuery .= ")";
}
```